src/frontend/components/Chat/SharedChatComponents/ChatMessageMetadatas/index.spec.tsx

```diff
@@ -0,0 +1,32 @@
+import { render, screen } from '@testing-library/react';
+import { DateTime } from 'luxon';
+import React from 'react';
+import { ChatMessageMetadatas } from '.';
+import { wrapInIntlProvider } from '../../../../utils/tests/intl';
+
+describe('<ChatMessageMetadatas />', () => {
+  it('renders the chatMessagesMetadatas and checks displayed informations and sizes are corrects.', () => {
```

Comment:
```suggestion
  it('renders the chatMessagesMetadatas and checks displayed information and sizes are correct.', () => {
```

Reply:
done in [058447a951165114aa40a1a78e5a26b7ff907df4](https://github.com/openfun/marsha/pull/1247/commits/058447a951165114aa40a1a78e5a26b7ff907df4)